practice-one/src/components/Account/Account.css

```diff
@@ -0,0 +1,24 @@
+.account {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 18px;
+    padding-bottom: 30px;
+}
+
+.account__avatar {
+    width: 45px;
+    height: auto;
+    border-radius: 12px;
+}
+
+.info__name,
+.info__desc {
+    font-size: var(--fs-xx-sm);
+    font-weight: var(--fw-sm);
+    line-height: normal;
+}
+
+.info__desc {
+    color: #99b2c6;
```

Comment:
did you define common color?

Reply:
I added it 
![image](https://github.com/datdragon777/React-Training/assets/74806826/5334b08c-ac82-4154-9cc7-5da1ff639545)